.github/ISSUE_TEMPLATE/FEATURE-REQUEST.yml

```diff
@@ -0,0 +1,47 @@
+name: Feature Request
+description: 🚨 Please make sure to fill out the template carefully. Otherwise, we may not be able to assist you. 🚨
+title: "[Feature]: "
+labels: ["✨ feature", "📋 needs review"]
+assignees:
+  - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to fill out this feature request. 🚀
+  - type: textarea
+    id: problem-description
+    attributes:
+      label: ✏️ Problem Description
+      description: Describe the problem or lack of functionality that you want to address. ✨
+      placeholder: What is not working as desired? Which feature is missing or could be improved?
+    validations:
+      required: true
+  - type: textarea
+    id: feature-request
+    attributes:
+      label: 💡 Feature Request
+      description: What feature do you want to request or improve? 🔍
```

Comment:
Suggestion: 'would you like to have improved'

Reply:
I am not sure if this is the right way to go. 
Some feature requests are not just improvements, but entirely new features. 
Suggestion: What feature should be improved or added?

Reply:
My suggestion was:
"What feature do you want to request or would like to have improved?"
Github did not mark the part of the sentence, I wanted to comment.
Your suggestion is also fine.

Reply:
Ah, okay, yes, I like your suggestion better. I had misunderstood that.